pkg/apis/deployer/terraform/v1alpha1/types_config.go

```diff
@@ -0,0 +1,45 @@
+// SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha1
+
+import (
+	lsv1alpha1 "github.com/gardener/landscaper/pkg/apis/core/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ManagedInstanceLabel describes label that is added to every terraform deployer managed resource
+// to define its corresponding instance.
+const ManagedInstanceLabel = "terraform.deployer.landscaper.gardener.cloud/instance"
+
+// ManagedDeployItemLabel describes label that is added to every terraform deployer managed resource
+// to define its source deploy item.
+const ManagedDeployItemLabel = "terraform.deployer.landscaper.gardener.cloud/deployitem"
```

Comment:
there is also a constants file in `pkg/deployers/terraform`. We should have only one place where constants are defined

Reply:
The constants there are specifics to the controller logic, should I move that to a `pkg/apis/deployer/terraform/constants.go` ?

Reply:
yes I that makes sense.
Then we have 1 file were all constants for the terraform deployer is defined